client/src/views/FightView.tsx

```diff
@@ -52,6 +52,8 @@ const FightView = () => {
     return [firstAd, secondAd];
   }, [language]);
 
+  const currentMapEffect = 0 + 1;
```

Comment:
You can check which background is used in the fight here instead, you have access to the fight variable